geoextent/__init__.py

```diff
@@ -1,3 +1,3 @@
 name = "geoextent"
 
-__version__ = '0.4.0'
+__version__ = '0.4.1'
```

Comment:
Since you're adding two new supported formats, this should be a minor version change, so please go with `0.5.0`.